ui/core.js

```diff
@@ -126,6 +128,67 @@ $.fn.extend( {
 				$( this ).removeAttr( "id" );
 			}
 		} );
+	},
+
+	form: function() {
```

Comment:
What I really care about here is the `form` variable. I don't care if we use `.prop()` or direct property access.